### PR TITLE
[Merged by Bors] - Set explicit resources on all containers

### PR DESCRIPTION
**Pull request by sbernauer**
# Description

Part of https://github.com/stackabletech/issues/issues/394
Supersedes https://github.com/stackabletech/issues/issues/394

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes
 
```[tasklist]
# Author
- [ ] Changes are OpenShift compatible
- [ ] CRD changes approved
- [ ] Helm chart can be installed and deployed operator works
- [ ] Integration tests passed (for non trivial changes)
```

```[tasklist]
# Reviewer
- [ ] Code contains useful comments
- [ ] (Integration-)Test cases added
- [ ] Documentation added or updated
- [ ] Changelog updated
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)
```

```[tasklist]
# Acceptance
- [ ] Feature Tracker has been updated
- [ ] Proper release label has been added
```

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

**Comment by sbernauer** [comment]
Tests passed :rocket:

File: docs/modules/zookeeper/pages/usage_guide/resource_configuration.adoc [hunk 1]
```diff
@@ -18,16 +18,18 @@ servers:
...
+
+A minimal HA setup consisting of 3 ZooKeeper instances has the following https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/[resource requirements]:
+
+* `600m` CPU request
+* `1800m` CPU limit
+* `1728m` memory request and limit
+* `750Mi` persistent storage
 
-If no resource requests are configured explicitly, the ZooKeeper operator uses the following defaults:
+Corresponding to the values above, the operator uses the following resource defaults:
```
[the remark this answers is not in the document]
Comment (sbernauer) [reply docs/modules/zookeeper/pages/usage_guide/resource_configuration.adoc:32]: As discussed offline we may document it at some later point of time

**Review by razvan — APPROVED** [review]
lgtm

**Comment by sbernauer** [comment]
bors r+
Thanks for review!

**Pull request closed without merging**